propolis/src/instance.rs

```diff
@@ -88,6 +108,7 @@ impl Instance {
         Ok(this)
     }
 
+    // TODO: Why is this separate from creating an instance?
```

Comment:
Initialization might take place over a longer period.  One could imagine in the dropshot version that individual disks and NICs are attached separately, in their own API calls.  I did not want to do it all in one go.